mc-server/start.sh

```diff
@@ -86,7 +86,12 @@ cd /usr/src/serverfiles/
 
 # Do that forever
 printf "%s\n" "Starting JAR file with: $RAM of RAM"
-java -Xms$RAM -Xmx$RAM -jar $JAR_FILE
```

Comment:
I just rewrote it below, don't worry I didn't delete the most important part of the file...